src/Layout/Tutorial/ConnectedTutorial.tsx

```diff
@@ -0,0 +1,17 @@
+import { RootState } from "reducers";
```

Comment:
Codacy found an issue: [Module 'reducers' is not listed as dependency in package.json](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5947793)